### PR TITLE
bpo-39379:sys.path[0] is already absolute path

### PR DESCRIPTION
**Pull request by ananthan-123**
https://bugs.python.org/issue39379

### CODE REVIEW

File: Doc/whatsnew/3.9.rst [hunk 1]
```diff
@@ -83,11 +83,10 @@ Other Language Changes
 
 * Python now gets the absolute path of the script filename specified on
   the command line (ex: ``python3 script.py``): the ``__file__`` attribute of
-  the :mod:`__main__` module and ``sys.path[0]`` become an
-  absolute path, rather than a relative path. These paths now remain valid
-  after the current directory is changed by :func:`os.chdir`. As a side effect,
-  a traceback also displays the absolute path for :mod:`__main__` module frames
-  in this case.
+  the :mod:`__main__` module become an absolute path, rather than a relative
```
Comment (tirkarthi) [inline Doc/whatsnew/3.9.rst]: Should it be "became"?

**Comment by tirkarthi** [comment]
NEWS entry is not needed for this change IMO.

**Comment by ananthan-123** [comment]
I have made the requested changes; please review again

**Comment by aeros** [comment]
@tirkarthi 
> NEWS entry is not needed for this change IMO.

Agreed, I'll add a `skip-news` label.

File: Doc/whatsnew/3.9.rst [hunk 2]
```diff
@@ -83,11 +83,10 @@ Other Language Changes
...
 * Python now gets the absolute path of the script filename specified on
   the command line (ex: ``python3 script.py``): the ``__file__`` attribute of
-  the :mod:`__main__` module and ``sys.path[0]`` become an
-  absolute path, rather than a relative path. These paths now remain valid
-  after the current directory is changed by :func:`os.chdir`. As a side effect,
-  a traceback also displays the absolute path for :mod:`__main__` module frames
-  in this case.
+  the :mod:`__main__` module became an absolute path, rather than a relative
+  path. These paths now remain valid after the current directory is changed
+  by :func:`os.chdir`. As a side effect, a traceback also displays the
```
Comment (aeros) [inline Doc/whatsnew/3.9.rst]:
```suggestion
  by :func:`os.chdir`. As a side effect, the traceback also displays the
```

In this context, "the traceback" makes more sense than "a traceback", IMO.

**Review by aeros — COMMENTED** [review]
Mostly LGTM, other than a minor point on grammar that was present in the original entry. Not critical, but I think we should address it.

Also, I think this should receive approval from @vstinner prior to merging, just to be sure (since he originally wrote the patch for this entry).

**Review by gvanrossum — APPROVED** [review]

**Comment by tirkarthi** [comment]
I have merged the PR since it's minor. I have left the issue open due to question at https://bugs.python.org/issue39379#msg360333 , Thanks for the patch and review.

**Pull request merged**